migrations/000035_AddTravelerBit.up.sql

```diff
@@ -0,0 +1,22 @@
+-- Copyright 2020 Google LLC
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--      http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+BEGIN;
+
+-- Traveler column is not indexed because we always iterate over keys by their created_at
+-- or revised_at time which is indexed and do secondary filtering in the server.
+ALTER TABLE Exposure
+    ADD COLUMN traveler BOOL NOT NULL DEFAULT false;
```

Comment:
Nit: it might be better to make this `is_traveler` to indicate its a boolean.

Reply:
our other bool fields don't have that prefix, but doesn't matter much to me.

Reply:
Ok nvm